src/components/dialog-full-screen/content.style.js

```diff
@@ -5,15 +5,29 @@ import contentClassicStyle from './content-classic.style';
 const StyledContent = styled.div`
   overflow-y: auto;
   padding: 0 16px;
-  @media screen and (min-width: 600px) {
-    padding: 0 24px;
-  }
-  @media screen and (min-width: 960px) {
-    padding: 0 32px;
-  }
-  @media screen and (min-width: 1260px) {
-    padding: 0 40px;
-  }
+
+  ${({ disableContentPadding }) => css`
+  
+    ${!disableContentPadding && css`
+      @media screen and (min-width: 600px) {
+        padding: 0 24px;
+      }
+      @media screen and (min-width: 960px) {
+        padding: 0 32px;
+      }
+      @media screen and (min-width: 1260px) {
+        padding: 0 40px;
+      }
+    `}
+
+    ${disableContentPadding && css`
```

Comment:
You don't need this `padding: 0` and the one below, one or the other will do

Reply:
Good point. I forgot to remove the one below